docs/reference-guide/modules/events/pages/event-processors/streaming.adoc

```diff
@@ -1180,8 +914,8 @@ Spring Boot::
 public class AxonConfig {
     // omitting other configuration methods...
     @Bean
-    public ConfigurerModule sequencingPolicyConfigurerModule(SequencingPolicy<EventMessage<?>> mySequencingPolicy) {
-        return configurer -> configurer.eventProcessing(
+    public ConfigurationEnhancer sequencingPolicyConfigurationEnhancer(SequencingPolicy<EventMessage<?>> mySequencingPolicy) {
+        return registry -> MessagingConfigurer.with(registry).eventProcessing(
```

Comment:
Yet again a wrong code sample using the non-existing `MessagingConfigurer#with` method.